<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditSecondNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Second Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 200);</string>
   </property>
   <property name="text">
    <string>TEST</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTestIt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButtonTestIt{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color: rgba(255, 255, 255, 210);
	border-radius: 5px;
}

QPushButton#pushButtonTestIt:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#pushButtonTestIt:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(150, 123, 111);

}</string>
   </property>
   <property name="text">
    <string>T E S T  I T</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>280</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 2px;</string>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(46, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>First Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFailOrSuccess">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>335</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 210);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>430</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>430</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelinfo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0, 0, 0, 210);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelBackground</zorder>
  <zorder>lineEditSecondNumber</zorder>
  <zorder>labelText</zorder>
  <zorder>pushButtonTestIt</zorder>
  <zorder>label</zorder>
  <zorder>lineEditFirstNumber</zorder>
  <zorder>labelFailOrSuccess</zorder>
  <zorder>labelinfo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
